components/Blog/blogList.js

```diff
@@ -2,10 +2,10 @@ import React from 'react'
 
 const BlogComponent = ({ content }) =>
   (<div className='blog-component pa4' >
-    <img src={content.imgUrl} />
+    <img src={content.imgUrl} className='mt4 mr4 mb2 ml0' />
     <div className='main-content'>
       <div className='title tl pa3 fw6 ttc'>{content.title}</div>
```

Comment:
This should ideally be an `h2` element for semantics (and the page should have an `h1` that describes the page). If the title is clickable, it should also be using a link to make it semantic and keyboard accessible.